R/write_sessioninfo.R

```diff
@@ -28,7 +28,13 @@
 #' @importFrom dplyr
 #' mutate
 #' select
+#'
 #' @export
+#'
+#' @examples
+#' \dontrun{
+#' write_sessioninfo(file = "./sessioninfo.txt")
```

Comment:
I suggest to simplify your example as `write_sessioninfo(file = "sessioninfo.txt")`

Reply:
Done in 4c0dc7a